tests_integration_with_local_chain/markets/omen/test_omen.py

```
@@ -389,13 +443,13 @@
     # Start by moving all funds from wxdai to xdai
     if initial_balances.wxdai > 0:
         WrappedxDaiContract().withdraw(
-            api_keys=test_keys,
+            api_keys=keys,
             amount_wei=xdai_to_wei(initial_balances.wxdai),
             web3=local_web3,
         )
 
     # Check that we have xdai funds, but no wxdai funds
-    initial_balances = get_balances(address=test_keys.bet_from_address, web3=local_web3)
+    initial_balances = get_balances(address=keys.bet_from_address, web3=local_web3)
```

Ruff

Undefined name `keys`

(F821)